code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -92,7 +100,7 @@ AI MODULES
 
 /******************** Safeguard ********************/
 /obj/item/aiModule/safeguard
-	name = "\improper 'Safeguard' AI module"
```

Comment:
The name will be treated as proper again, so examining the board would show "This is Safeguard AI module." -- same for the others below